### PR TITLE
adds anyninesDark and bosch theme

### PR DESCRIPTION
**Pull request by r3p0x**

### CODE REVIEW

File: src/theme/ThemeProvider.tsx [hunk 1]
```diff
@@ -26,12 +28,18 @@ export function ThemeProvider(props: ThemeProviderProps): JSX.Element {
...
+    if (theme === 'anynines-dark') {
+      return anyninesDark
+    }
     if (theme === 'toyota') {
       return toyota
     }
     if (theme === 'thomsit') {
       return thomsit
     }
+    if (theme === 'bosch') {
```
Comment (anynines-johannchopin) [inline src/theme/ThemeProvider.tsx:40]: Not to implement yet but if the number of themes grow and grow you could use a `THEMES` object like:

```js
const THEMES:  {[key in Theme]: ThemeOptions } = {
  anynines,
  'anynines-dark': anyninesDark,
  // ...
}
```

and then update the `getTheme` function to:

```ts
function getTheme(): ThemeOptions {
  return THEMES[props.theme]
}
```

Like that you don't need to write extra if statement for each themes 👍

**Review by anynines-johannchopin — APPROVED** [review]
CR ✅  Thanks for the long weekend 😆

File: src/theme/ThemeProvider.tsx (hunk 1 above)
Comment (r3p0x) [reply src/theme/ThemeProvider.tsx:40]: That's a great idea <3  https://anynines.atlassian.net/browse/CHAP-1408 in case you wanna do it right away :)

Comment (anynines-johannchopin) [reply src/theme/ThemeProvider.tsx:40]: Sure I should have time for this little one 👍

**Pull request merged**